### ISSUE TITLE
Show "trending" live thread on frontpage

### ISSUE THREAD

**Issue by Nxt3** [ccrama/Slide#1964]
On the Desktop--a (popular?) live thread will show up on your frontpage even if you aren't sub'd to the subreddit that is hosting that live thread.

For example, [/r/worldnews is hosting a livethread currently](https://www.reddit.com/live/xatg2056flbi)--but I'm not sub'd to that, so I don't see it.

Desktop:
![image](https://cloud.githubusercontent.com/assets/2940826/17066048/3acd3bc6-5013-11e6-882b-a4b6afcc5265.png)

I think Slide should do the same thing.

Edit: Actually even weirder--is that if I go to /r/worldnews, the live thread won't be anywhere. I can only see it at the top of the frontpage if I am logged out (as a Guest) in Slide.

**Comment by Alexendoo** [comment]
I remember looking at this to try and see where it came from because it would've been useful (before we had actual live thread support, would be even better now), couldn't find anything in the API so looked about a bit to see if maybe we could use the same algorithm for choosing the promoted /r/live thread as them. However it's hand picked by admins manually it seems https://github.com/reddit/reddit-plugin-liveupdate/blob/master/reddit_liveupdate/controllers.py#L1147-L1171

The only place it seems to be included is in the HTML response in reddit's' desktop site itself, as far as I could find anyway

Also I thought I was experiencing the same thing as you with the worldnews thread but turns out I had just hidden it on my account because it was rather depressing

**Comment by Nxt3** [comment]
I figured there wouldn't be anything in the API. That's a shame. Is there a way to parse the HTML response? That seems like so much work though--so maybe it isn't important. Just thought it would be cool.

I didn't hide it because I first saw the live thread on desktop--then went over to Slide and it was no where to be found.

**Comment by Alexendoo** [comment]
Yeah it could be done but it's not a small request, would have a pretty large overhead to download. It also wouldn't have any garuntees of staying stable like the API does

**Comment by ccrama** [comment]
Agree with @Alexendoo . I think this is closable

**Comment by Nxt3** [comment]
wah wahhh. :( I'll make a request [on their GitHub](https://github.com/reddit/reddit/issues/1654). Doubt anything will come of it, but we can hope!

**Comment by Nxt3** [comment]
See https://github.com/reddit/reddit-plugin-liveupdate/pull/138

**Comment by 13steinj** [comment]
Welp, @ccrama, I was right in my last statement. https://github.com/reddit/reddit-plugin-liveupdate/pull/139 should be in soon enough though.

**Comment by ccrama** [comment]
@13steinj that is unfortunate, but glad it's getting implemented! Thank you for the heads up and your work on the Reddit API :)

**Comment by marwanad** [comment]
Is this still up for grabs? This is one interesting feature to have. It could simply be an extra CardView added with a smaller size than the submissions view one.

**Comment by ccrama** [comment]
That would be quite cool @marwanad, definitely up for grabs if you're interested in implementing it! I could see it being a little complicated to implement, but it could be added to the [SubmissionAdapter](https://github.com/ccrama/Slide/blob/master/app/src/main/java/me/ccrama/redditslide/Adapters/SubmissionAdapter.java) which is updated [here](https://github.com/ccrama/Slide/blob/master/app/src/main/java/me/ccrama/redditslide/Fragments/SubmissionsView.java#L451)

**Comment by marwanad** [comment]
@ccrama Awesome! I'll give it a shot and post on IRC if I come across any issues.
